### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by asdacosta**
## Because

Set default content

## This PR

* Set default content

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/asynchronous_javascript_and_apis/asynchronous_code.md [hunk 1]
```diff
@@ -73,15 +73,15 @@ Of course, there are many more occasions where one would want to use Promises be
 <div class="lesson-content__panel" markdown="1">
 
 1. Read this [Promises article](https://davidwalsh.name/promises). It's a good starting place and it's short and to the point.
-2. Watch this [Promises video](https://youtu.be/DHvZLI7Db8E).  It's a good place to get a feel for how one might actually use promises in the wild.
-3. Watch this [What is Event Loop? video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to understand how asynchronous code works in JavaScript.
-4. Read [promise basics](https://javascript.info/promise-basics), which is an excellent resource for beginners.
+1. Watch this [Promises video](https://youtu.be/DHvZLI7Db8E).  It's a good place to get a feel for how one might actually use promises in the wild.
+1. Watch this [What is Event Loop? video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to understand how asynchronous code works in JavaScript.
+1. Read [promise basics](https://javascript.info/promise-basics), which is an excellent resource for beginners.
```
Comment (ManonLef) [inline javascript/asynchronous_javascript_and_apis/asynchronous_code.md:78]:
```suggestion
1. Watch this [video about promises](https://youtu.be/DHvZLI7Db8E).  It's a good place to get a feel for how one might actually use promises in the wild.
1. Watch this [video titled "What is Event Loop?"](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to understand how asynchronous code works in JavaScript.
1. Read this [promise basics article](https://javascript.info/promise-basics), which is an excellent resource for beginners.
```

File: javascript/asynchronous_javascript_and_apis/asynchronous_code.md [hunk 2]
```diff
@@ -90,8 +90,8 @@ This section contains questions for you to check your understanding of this less
...
-3. Read [Chapter 2: Callbacks](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch2.md) and [Chapter 3: Promises](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch3.md) from "You Don't Know JavaScript". Both chapters dive deep into their respective topics, which may make for a difficult - but worthwhile - read.
+- The [Promises by ColorCode](https://www.youtube.com/watch?v=TnhCX0KkPqs) is a nice introduction to Promises if you need additional information.
+- For a more interactive explanation and example, try a [Scrim on asynchronous coding](https://scrimba.com/scrim/cof4e4fb797a2d0a236ea38ce?embed=odin,mini-header,no-next-up).
+- Read [Chapter 2: Callbacks](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch2.md) and [Chapter 3: Promises](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch3.md) from "You Don't Know JavaScript". Both chapters dive deep into their respective topics, which may make for a difficult - but worthwhile - read.
```
Comment (ManonLef) [inline javascript/asynchronous_javascript_and_apis/asynchronous_code.md:97]:
```suggestion
- The [Promises by ColorCode video](https://www.youtube.com/watch?v=TnhCX0KkPqs) is a nice introduction to Promises if you need additional information.
- For a more interactive explanation and example, try a [Scrim on asynchronous coding](https://scrimba.com/scrim/cof4e4fb797a2d0a236ea38ce?embed=odin,mini-header,no-next-up).
- Read ["You Don't Know JavaScript" Chapter 2: Callbacks](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch2.md) and [Chapter 3: Promises](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch3.md). Both chapters dive deep into their respective topics, which may make for a difficult - but worthwhile - read.
```

**Review by ManonLef — CHANGES_REQUESTED** [review]
thanks again, I added some descriptive link suggestions while we're here. Let me know what you think :)

**Comment by asdacosta** [comment]
> thanks again, I added some descriptive link suggestions while we're here. Let me know what you think :)

Perfect

**Review by ManonLef — APPROVED** [review]

**Pull request merged**